tests/test-installer

```diff
@@ -40,3 +40,16 @@ test "$?" = "0" || exit 1
 grep "myregistry.io" $cfgfile || exit 1
 grep "auth-file.json" $cfgfile || exit 1
 command -v crucible || exit 1
+
+# identity
+touch /tmp/auth-file.json || true
+cfgfile=$(grep SYSCONFIG= crucible-install.sh | cut -d '=' -f2 | sed 's/"//g')
+idfile=$(grep IDENTITY= crucible-install.sh | cut -d '=' -f2 | sed 's/"//g')
+# TODO(rfolco): temporary workaround until we make it distro generic
+sudo mkdir -p $(dirname $cfgfile)
+sudo ./crucible-install.sh \
+  --registry myregistry.io/crucible \
+  --auth-file /tmp/auth-file.json \
+  --name "Nobody" \
+  --email "nobody@somewhere.com"
```

Comment:
Unless I'm not understanding things I don't think this worked in the CI test.  It says the test passed, but I don't see these values in the following grep output:

https://github.com/perftool-incubator/crucible/pull/140/checks#step:3:128